timemachine/fe/rbfe.py

```diff
@@ -562,6 +569,24 @@ def estimate_relative_free_energy(
 
 
 def run_pair(mol_a, mol_b, core, forcefield, protein, n_frames, seed, n_eq_steps=10000):
+
+    # vacuum leg
+    vacuum_host_config = None
+    vacuum_res = estimate_relative_free_energy(
+        mol_a,
+        mol_b,
+        core,
+        forcefield,
+        vacuum_host_config,
+        seed,
+        n_frames=n_frames,
+        prefix="vacuum",
+        n_eq_steps=n_eq_steps,
+    )
+
+    # vacuum error should be below a threshold, otherwise this edge is likely problematic
+    assert np.linalg.norm(vacuum_res.all_dGs) < 1.0
```

Comment:
Curiosity: Has the false positive rate of this assertion been checked in some way? (Intuitively this looks likely to introduce spurious assertion errors.)

Clarification: Is this assertion intended to reference the estimated dGs or the estimated errors? 

```suggestion
    # vacuum error should be below a threshold, otherwise this edge is likely problematic
    assert np.linalg.norm(vacuum_res.all_errs) < 1.0
```